### PR TITLE
Added Haxe

### PR DESCRIPTION
**Pull request by KOLANICH**
- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md

I am:
  - [ ] Suggesting edits to an existing Topic page
  - [x] Curating a new Topic page

***********EDITING AN EXISTING TOPIC PAGE************

I'm suggesting these edits to an existing topic:
- [ ] Image (and my file is `*.png`, square, dimensions 288x288)
- [ ] Content (and my changes are in `index.md`)

Please explain why these changes are necessary:

************CURATING A NEW TOPIC PAGE************

- [x] I've formatted my changes as a new folder directory, named for the topic as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]`)
- [ ] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

Please explain why you think this Topic page should be curated:

It's a programming language.

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

### CODE REVIEW

File: topics/haxe/index.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
+---
+aliases: hx, hxml
+created_by: Nicolas Cannasse, Haxe Foundation
+display_name: Haxe
+github_url: https://github.com/HaxeFoundation
+released: '2005'
+short_description: A metalanguage resembling ECMAScript which can be transpiled into a variety of languages
```
Comment (MikeMcQuaid) [inline topics/haxe/index.md]: JavaScript?

File: topics/haxe/index.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+topic: haxe
+url: https://haxe.org/
+wikipedia_url: https://en.wikipedia.org/wiki/Haxe
+related: es6, python, java, actionscript, c-plus-plus, csharp, php, lua
+---
+A language resembling ECMAScript much which can be transpiled into ActionScript 3, ECMAScript, Java, C++, C#, PHP, Python and Lua.
```
Comment (MikeMcQuaid) [inline topics/haxe/index.md]: - JavaScript?
- Remove `much`

**Comment by MikeMcQuaid** [comment]
@KOLANICH Thanks for the topic! Can you fix up the failing tests: https://travis-ci.org/github/explore/builds/300437780?utm_source=github_status&utm_medium=notification

File: topics/haxe/index.md (hunk 1 above)
Comment (KOLANICH) [reply topics/haxe/index.md]: >The Haxe Programming Language was designed to be simple yet powerful. Its syntax largely follows the ECMAScript standard, but deviates where necessary.

File: topics/haxe/index.md (hunk 2 above)
Comment (KOLANICH) [reply topics/haxe/index.md]: OK.

**Comment by KOLANICH** [comment]
BTW one of tests is wrong: `ActionScript 3` doesn't pass it.

**Comment by MikeMcQuaid** [comment]
Thanks for your first contribution to github/explore, @KOLANICH! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request merged**